src/appsco-access-report-page.html

```diff
@@ -92,27 +53,6 @@
             :host .flex {
                 @apply(--layout-flex);
             }
-            :host([tablet-screen]) {
```

Comment:
media-queries are needed for resource section. Simply replace info with resource-width.